ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -19,16 +19,28 @@
  * {@link Router#getBlob(String, GetBlobOptions, Callback)} call.
  */
 public class GetBlobOptions {
+  public static final GetBlobOptions DEFAULT_OPTIONS = new GetBlobOptions(OperationType.All, null);
```

Comment:
There is nothing special about this case though, if you think about it. We could even have `GETBLOBINFO_OPTIONS`, `GETBLOBDATA_NORANGE_OPTIONS`, etc. pre-created as static final objects and reused by that argument. I am sure there are several other classes and objects where this would apply too - a lot of exceptions that are thrown, for instance.

I don't think it would make a perceivable difference and compilers may even be smart enough to do many such optimizations anyway.